metalearners/drlearner.py

```diff
@@ -398,3 +405,44 @@ def _pseudo_outcome(
             )
 
         return pseudo_outcome
+
+    @classmethod
+    def necessary_onnx_models(cls) -> set[str]:
+        return {TREATMENT_MODEL}
+
+    @copydoc(MetaLearner.build_onnx, sep="")
+    def build_onnx(self, models: Mapping[str, Sequence], output_name: str = "tau"):
+        """In the DRLearner case, the necessary models are:
+
+        * ``"treatment_model"``
+        """
+        warning_experimental_feature("build_onnx")
+        check_onnx_installed()
+        check_spox_installed()
+        import spox.opset.ai.onnx.v21 as op
+        from onnx.checker import check_model
+        from spox import Var, build, inline
+
+        self._validate_feature_set_none()
+        self._validate_onnx_models(models, self.necessary_onnx_models())
+
+        input_dict = infer_input_dict(models[TREATMENT_MODEL][0])
+
+        treatment_output_name = models[TREATMENT_MODEL][0].graph.output[0].name
```

Comment:
Maybe some explicit validation that the models meet your API expectations?

Reply:
It is done in `_validate_onnx_models`

Reply:
🙈